distributed/dashboard/components/shared.py

```diff
@@ -310,7 +310,7 @@ async def cb():
                 metadata = await self.server.get_profile_metadata()
             else:
                 metadata = None
-            if isinstance(prof, gen.Future):
```

Comment:
There are a lot of cases like this where we're renaming tornado things that probably shouldn't be renamed.